### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by SaidRasinlic**
### In this milestone, I've done the following things:

- Deleted all the boilerplate from CRA
- Created directory named "components"
- Created "Calculator.js" file inside "components" 
- Inside "Calculator.js" created the UI requested design

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (CollinsTatang) [inline src/App.js]: - [x] Please your `App.js` component should be class-based and not function base.

```javascript
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      <button className="btn" type="button">3</button>
+      <button className="btn orange" type="button">+</button>
+
+      <button className="btn span-2" type="button">0</button>
+      <button className="btn" type="button">.</button>
+      <button className="btn orange" type="button">=</button>
+
+    </div>
+  );
+}
```
Comment (CollinsTatang) [inline src/components/Calculator.js]: - [x] Please your `Calculator.js` component should be class-based and not function base.

```javascript
class Calculator extends Component {
  render() {
    // reference as a js object
    return {
      <div>
       // Calculator app design goes here
      </div>
    };
  }
}
```

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @SaidRasinlic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (SaidRasinlic) [reply src/App.js]: Done

File: src/components/Calculator.js (hunk 2 above)
Comment (SaidRasinlic) [reply src/components/Calculator.js]: Done, although I couldn't see it writes to use Class Components instead of Function Components 😅

**Review by Lordkaito — APPROVED** [review]
Hi @SaidRasinlic, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**